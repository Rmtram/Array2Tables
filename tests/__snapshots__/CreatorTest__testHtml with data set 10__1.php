<?php return '<table><thead><th>text</th></thead><tbody><tr><td>a</td></tr><tr><td>b</td><td>c</td></tr></tbody></table>
';
