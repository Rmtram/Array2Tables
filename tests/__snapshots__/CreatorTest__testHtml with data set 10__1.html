<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<html><body>
<table>
<thead><th>text</th></thead>
<tbody>
<tr><td>a</td></tr>
<tr>
<td>b</td>
<td>c</td>
</tr>
</tbody>
</table>
</body></html>
